src/TheThingsNetwork.cpp

```diff
@@ -411,6 +340,78 @@ void TheThingsNetwork::configure_channels() {
       break;
     default:
       debugPrintLn("Invalid frequency plan");
+      break;
+  }
+  modemStream->println(F("sys reset"));
+  String version = readLine(3000);
+  if (version == "") {
+    debugPrintLn(F("Invalid version"));
+    return;
+  }
+  model = version.substring(0, version.indexOf(' '));
+  debugPrint(F("Version is "));
+  debugPrint(version);
+  debugPrint(F(", model is "));
+  debugPrintLn(model);
+  int dr = -1;
+  String str = "";
+  if (model == F("RN2483")) {
```

Comment:
Don't check on the `model`. We know what to configure based on the frequency plan